app/components/Signin/index.jsx

```diff
@@ -46,17 +51,27 @@ var Login = React.createClass({
             });
         });
     },
+
+    _quick_login(){
```

Comment:
Login Page redirected to Genius Center. Please remember add "app_id" parameter(IMPORTANT).